### DISCUSSION TITLE
Configure appearance and function of homepage link

### DISCUSSION THREAD

**Discussion by MercuryChaos** [requarks/wiki#6443] [Help / Questions]
Hi, I'm pretty new at using wiki.js, and I want to know if it's possible to reconfigure the homepage link in the navigation sidebar so that it points somewhere else and uses a different image.
![Screen Shot 2023-05-21 at 1 50 51 PM](https://github.com/requarks/wiki/assets/3300217/9d9bec15-23bf-4faa-88e0-5e4c6f676ac8)

I'm working on a resource wiki for the transgender community, and I want to have a "quick exit" link that's always visible so that people using the wiki can quickly leave the page if someone comes up behind them that they don't want to out themselves to. The Home button *would* be perfect for this if I could change the appearance and make it point to Google.com or something, but that doesn't seem to be something that can be configured in the wiki settings. Is this possible, or is there a better way to accomplish the thing that I'm trying to do?

**Comment by fnafox** [comment]
I don't believe this is possible to do. Maybe it could be accomplished with some custom scripts but I can't help you there.
However, if you're not set on using the Site Tree navigation, you could very easily make a permanently accessible quick exit link if you stick to Static navigation.
